### ISSUE TITLE
[AutoModpack] Error! Something went wrong! - cant connect to server 1.19.2

### ISSUE THREAD

**Issue by frizzbees** [Skidamek/AutoModpack#109]
**Describe the bug**

using the latest 1.19.2 version available through modrinth on both the client and the server, client will succsesfully download jar files and configs from the server but will eventually error out showing 

```
[AutoModpack] Error! Something went wrong!

Critical error while downloading modpack.
"com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 15 column 2 path $"
More details in logs
```

**To Reproduce**
1. install automodpack on both client and server
2. connect to the server on the client
3. wait until jars are downloaded and then error occurs

**Expected behavior**

expected mods to download and prompt player for restart instead of erroring out

**Screenshots**

**AutoModpack and Minecraft Version you are using (please complete the following information):**
 -  3.1.0 (from https://modrinth.com/mod/automodpack/version/GHoryMSM)
 - 1.19.2 fabric 

**Additional context**

**Client & Server log**
client log: https://mclo.gs/ByWcOQT
server log: https://mclo.gs/mrLkHgv

**Comment by frizzbees** [comment]
its worth noting that taking the jars from the automodpack/modpacks/(server) directory and dragging them into the mods folder allows me to connect to the server, so the jars are downloading fine

**Comment by Skidamek** [comment]
Thanks for reporting, i got which part of code is breaking something, it didn't happen to me before and it is related to the mods you are specifically using. It won't yet fix the problem but it will tell me where i can find the cause. I added more debugging logs to the related code. Could you just replace the automodpack jars (it is compressed in zip) and try to reproduce the issue, when you get it please send me new logs :)
[unzip-this-file.zip](https://github.com/Skidamek/AutoModpack/files/11232665/unzip-this-file.zip)

**Comment by frizzbees** [comment]
hey ! thanks for the quick response

heres both log files with the new JAR

client: https://mclo.gs/WorWPuu
server: https://mclo.gs/u1co5j0

**Comment by Skidamek** [comment]
Thank you, now we know it is `HopoBetterRuinedPortals-[1.19-1.19.3]-1.3.3.jar`
I checked what is wrong with it and the fabric.mod.json file is wrongly formatted, which throws exceptions while trying to parse it as a json. The issue is already reported on author's discord but it is not fixed yet.

Here you have quickly fixed json in they mod by me it should now work completely fine:
[FIXED-ZIPED-HopoBetterRuinedPortals-[1.19-1.19.3]-1.3.3.zip](https://github.com/Skidamek/AutoModpack/files/11232830/FIXED-ZIPED-HopoBetterRuinedPortals-.1.19-1.19.3.-1.3.3.zip)

![image](https://user-images.githubusercontent.com/67871298/232045958-f1cd55c5-2f19-4609-a19b-4e1ff0ec7633.png)

**Comment by frizzbees** [comment]
this seemed to fix it! thank you so much, closing this issue now
